api/v1alpha1/common_types.go

```diff
@@ -60,20 +60,24 @@ const (
 )
 
 type SyncthingPeer struct {
-	// TCP address of the Syncthing peer
+	// Address Is an address which is used to connect to the other Syncthing peer.
 	Address string `json:"address"`
-	// Syncthing ID of the peer
+	// ID Is the peer's Syncthing ID.
 	ID string `json:"ID"`
-	// Introducer flag determines whether this peer should
-	// introduce us to other peers sharing this volume
+	// Introducer is a flag which determines whether this peer should
+	// introduce us to other peers sharing this volume.
 	Introducer bool `json:"introducer"`
 }
 
 type SyncthingPeerStatus struct {
-	// TCP address of the Syncthing peer
+	// Address Represents the TCP address of the Syncthing peer.
 	Address string `json:"address"`
-	// Syncthing ID of the peer
+	// ID Is the peer's Syncthing ID.
 	ID string `json:"ID"`
-	// Whether the peer is currently connected
+	// Connected Is a flag indicating whether peer is currently connected.
 	Connected bool `json:"connected"`
+	// IntroducedBy Is the ID of the peer that introduced this peer.
+	IntroducedBy string `json:"introducedBy,omitempty"`
+	// DeviceName Is a friendly name for the given device.
+	DeviceName string `json:"deviceName,omitempty"`
```

Comment:
Currently, the `Name` is the hostname of the device where the configured Syncthing instance is running, when introduced. At the moment, this defaults to the Syncthing's Pod name; however, this is something we can improve in the future so that a list of peers can appear as "us-east-1-backup-ns,"  "analytics-backend," etc.